snorkel/classification/snorkel_classifier.py

```diff
@@ -371,22 +384,36 @@ def score(self, dataloaders: List["DictDataLoader"]) -> Dict[str, float]:
         self.eval()
 
         metric_score_dict = dict()
+        # By default, evaluate all labels on corresponding task of same name
 
         for dataloader in dataloaders:
             results = self.predict(dataloader, return_preds=True)
-            for task_name in results["golds"].keys():
-                metric_scores = self.scorers[task_name].score(
-                    results["golds"][task_name],
-                    results["preds"][task_name],
-                    results["probs"][task_name],
+
+            # By default, evaluate each label_name on task with corresponding task_name
+            eval_dict = {name: name for name in results["golds"].keys()}
+            # Then, override the current results with manually specified mappings
+            eval_dict.update(remap_labels)
+
+            for label_name, task_name in eval_dict.items():
+                # Use the original gold labels, which include abstains
+                golds = (
+                    dataloader.dataset.Y_dict[  # type:ignore
+                        label_name
+                    ]
+                    .numpy()
+                    .squeeze()
                 )
+                preds = results["preds"][task_name]
+                probs = results["probs"][task_name]
+                metric_scores = self.scorers[task_name].score(golds, preds, probs)
```

Comment:
kwargs here? have seen it elsewhere

Reply:
the `Scorer.score` doesn't accept kwargs: https://github.com/HazyResearch/snorkel/blob/redux/snorkel/classification/scorer.py#L53

Reply:
i mean `golds=golds, preds=preds...`

Reply:
got it!